client/src/pages/Home.js

```diff
@@ -1,20 +1,47 @@
 import React from "react";
-import { Link } from "react-router-dom";
+import { getAccount } from "../apiAccount/fetchAccount";
+import Countdown from "../components/countDown/CountDown";
 import Header from "../components/Header/Header";
+import UserDataDate from "../components/userData/UserDataDate";
+import UserDataNames from "../components/userData/UserDataNames";
+import useAsync from "../hooks/useAsync";
+import bgImageSrc from "../assets/homeBgImage.png";
+import styled from "@emotion/styled";
 
 export default function Home() {
+  const { data: account, error, loading } = useAsync(getAccount);
+
   return (
-    <div>
+    <HomeContainer>
       <Header />
-      Home (in Arbeit)
-      <ul>
-        <li>
-          <Link to="/todos">ToDo Liste</Link>
-        </li>
-        <li>
-          <Link to="/account">Create Account</Link>
-        </li>
-      </ul>
-    </div>
+      {error && <p>Error</p>}
+      {loading && <p>Loading ...</p>}
+      {account && <UserDataNames account={account} />}
+      <section>{account && <Countdown date={account.date} />}</section>
```

Comment:
You could put the section in the expression, so the section would not be empty, if account is not there.